test/Utils.geometryUtils.test.js

```diff
@@ -42,4 +42,6 @@ describe( 'hasGroupGaps', () => {
 
 	} );
 
+	// SHOULD WE ADD NEW TEST WITH RANGE?
+
```

Comment:
I think it would be nice to add one or two more tests here that check the group gaps with a custom range